src/plugins/intel_gna/src/backend/am_intel_dnn.cpp

```diff
@@ -1757,7 +1758,7 @@ void GNAPluginNS::backend::AMIntelDNN::WriteInputAndOutputText() {
         if (numItems) {
             auto rmse = sqrt(summOfSqDiff / numItems);
             auto avg = summOfDiff / numItems;
-            std :: cout << std::left << std::setw(55) << out_file_name.str()
+            ov::intel_gna::log::trace() << std::left << std::setw(55) << out_file_name.str()
```

Comment:
I just wonder if modifications of the output stream flags are permanent and would decrease readability of further logs

Reply:
no, they are not permanent, please take a look at the documentation - https://en.cppreference.com/w/cpp/io/manip/setw